resources/views/pages/home.ejs

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <% include ../partials/head %>
+    <link rel="stylesheet" href="/resources/css/home.css" />
+  </head>
+  <body class="container">
+    <header>
+      <% include ../partials/header %>
+    </header>
+
+    <main class="home">
+      <div class="offer-banner">
+        <% for(var i=0; i < offers.length; i++) { %>
+        <div class="mySlides fade">
+          <img
+            src="<%= offers[i].bannerImageUrl %>"
+            alt="<%= categories[i].bannerImageAlt %>"
+            class="offer-banner__img"
+          />
+        </div>
+        <% } %>
+
+        <a class="prev">PREV</a>
+        <a class="next">NEXT</a>
+        <div class="dot-container">
+          <% for(var i=0; i < offers.length; i++) { %>
+          <span class="dot"></span>
+          <% } %>
+        </div>
+      </div>
+
+      <% for(var i=0; i < categories.length; i++) { %>
+      <div class="category">
+        <img src="<%= categories[i].imageUrl %>" alt="<%= categories[i].name %> logo" class="category-logo" />
+        <div class="flex-column category-detail">
+          <h1 class="category-detail__main"><%= categories[i].name %></h1>
+          <h3 class="category-detail__sub"><%= categories[i].description %></h3>
```

Comment:
No H2 on the page. Don't use skip level headings.

Reply:
done